sonar-ws/src/main/java/org/sonarqube/ws/client/project/ProjectsService.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2016 SonarSource SA
+ * mailto:contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+package org.sonarqube.ws.client.project;
+
+import org.sonarqube.ws.client.BaseService;
+import org.sonarqube.ws.client.PostRequest;
+import org.sonarqube.ws.client.WsConnector;
+
+/**
+ * Maps web service {@code api/projects}.
+ * @since 5.5
+ */
+public class ProjectsService extends BaseService {
+
+  public ProjectsService(WsConnector wsConnector) {
+    super(wsConnector, "api/projects");
+  }
+
+  /**
+   * Provisions a new project. This method does not allow to specify the parameter {@code branch}.
+   *
+   * @param projectKey the non-null key of project
+   * @param projectName the non-null name of project
+   * @throws org.sonarqube.ws.client.HttpException if HTTP status code is not 2xx.
+   */
+  public void create(String projectKey, String projectName) {
```

Comment:
this is the pattern used for all other WS. See [ActivityWsRequest](https://github.com/SonarSource/sonarqube/blob/bf27baa7ef13bef1a1296c1c09b845580e65be44/sonar-ws/src/main/java/org/sonarqube/ws/client/ce/ActivityWsRequest.java#L27-L27), one [SearchWsRequest](https://github.com/SonarSource/sonarqube/blob/f5d3461ffd3c5936a3bdc0999f6a9f71b9f10cd2/sonar-ws/src/main/java/org/sonarqube/ws/client/component/SearchWsRequest.java#L28-L28) for examples. You'll fine dozen of other examples in the ws client code. My point is I prefer to have the same pattern everywhere and we know where to find the information when needed.
Having the *WsRequest used in the Action gives a good way to validate the class, as it's well testeed